conduce.py

```diff
@@ -10,7 +10,27 @@ def list_from_args(args):
     object_to_list = args.object_to_list
     del vars(args)['object_to_list']
 
-    return api.list_object(object_to_list, **vars(args))
+    if object_to_list.lower() == "lenses" or object_to_list.lower() == "templates":
+        object_to_list = "LENS_TEMPLATE"
+
+    return api.list_resources(object_to_list.upper().rstrip('S'), **vars(args))
+
+
+def find_resource(args):
+    if args.type is not None:
+        resource_type = args.type
+
+        if resource_type.lower() == "lenses" or resource_type.lower() == "templates":
```

Comment:
nitpick: we may instead make this raise an exception that returns the valid list of resource types

Reply:
If there is an API to get that list, maybe.  Otherwise we don't want to maintain that here.

Besides we do raise in that case.  We display the error message returned by the server

```
Kevins-MacBook-Pro:conduce-python-api kvl$ conduce-api list otherthings
400 Client Error: Bad Request for url: https://dev-app.conduce.com/conduce/api/v2/resources/searches
Could not read json request: cannot unmarshal input

```

:/